### DISCUSSION TITLE
Show leaderboard on the result screen only if the play is rankable and in top50

### DISCUSSION THREAD

**Discussion by cihe13375** [ppy/osu#13151] [Ideas]
**Describe the new feature:** 
Now there's a leaderboard shown on the result screen, comparing the player's score with top50. However, imo there're two problems in this design currently:

1)The majority of players rarely get into top50 of the leaderboard. In most time they get something like #(3748). In this case it's nonsense to list their scores along with top50 and compare them because the difference is so large (except "telling" players they suck at this game).

2)The leaderboard still shows even if the play is unranked (relax/autopilot/etc.). This doesn't make sence either since the play is, well, unranked.

**Proposal designs of the feature:** 
imo there're two possible fixes for this:
1) Show leaderboard only if the play is rankable and in top50, as the title says.
2) Add an option to hide leaderboard on the result screen. It can be a button on the result screen, and it'll remember user's last choice.

I'd NOT recommend solve issue (1) by showing leaderboard "around" current play instead (like, if I'm #(3750), then show #(3725)~#(3775)), because I don't think players care who is 1 rank higher or lower than him (or smth like this).

**Comment by bdach** [comment]
I think the way that you "don't recommend" is the direction we'll be taking, actually. That's how it already works in timeshift.

Not sure about unranked mods, yet. Either hiding or scoping leaderboard down to only plays with the same unranked mods (if supporter) seem like sane starting choices to me.
